src/components/Dropdown/Dropdown.js

```diff
@@ -11,7 +11,7 @@ const Item = styled.div`
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
-  padding: 0 24px;
```

Comment:
I prefer 24. This also matches the `<Input />` component.

Reply:
The consensus in Slack seems to be less LR padding. Please make the `<Input />` match. I probably should have just set that in the theme somewhere.